orchestrator/cli/generator/generator/helpers.py

```diff
@@ -44,10 +49,117 @@ def get_product_block_file_name(product_block: dict) -> str:
     return get_product_block_variable(product_block)
 
 
+def get_existing_product_blocks() -> dict[str, Any]:
```

Comment:
move product block helpers to its own file for shorter files and a bit more clean